src/rpdk/core/data/pytest-contract.ini

```diff
@@ -1,8 +1,11 @@
 [pytest]
+log_cli = True
```

Comment:
can we add a parameter to allow people to get a more verbose error message if they want

Reply:
`--verbose` parameter is there. if you mean traceback printing, it's set to short which outputs only assertion messages and associated methods; I am not sure what do we get from printing out full traceback but might be a good idea to give an option;